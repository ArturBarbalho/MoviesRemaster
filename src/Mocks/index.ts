import { BannerProps, CardProps, bigbannerProps } from "../typesProps";
export const BiggerBannerMock:bigbannerProps = {
  bigsrc: 'https://image.tmdb.org/t/p/original//9VIURlZvfnmFACs2m0MdUDDC3In.jpg',
  smallsrc:'https://image.tmdb.org/t/p/original//ifRFLx83Xk1DcwAS3OScgI6HmWO.jpg',
  genres:['Action', 'Comedy'],
  id:2,
  media:'movie',
  overview:'Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis...',
  title:'ONYANKOPON 3 O IN√çCIO'
}
export const CardMock:CardProps = {
  id:2,
  media:'movie',
  src:'http://www.claquete.com/fotos/filmes/poster/12861_medio.jpg',
  title:'ONYANKOPON 2 O RETORNO'
}

export const BannerMock:BannerProps = {
    media: 'movie',
    relaseDate: '(1800)',
    id:1,
    title:'ONYANKOPON',
    src:'https://image.tmdb.org/t/p/original//9VIURlZvfnmFACs2m0MdUDDC3In.jpg',
    overview:'Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis...',
    genres: ['Action', 'Comedy' ]
  }