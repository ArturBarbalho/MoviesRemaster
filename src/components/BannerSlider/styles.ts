import styled, { css } from 'styled-components'
type props = {
  left:string
  right:string
}
export const Wrapper = styled.main<props>`
${({left, right})=>css`
height: 100vh;
overflow-x: hidden ;
 .slick-dots {
      list-style: none;
      display: flex !important;
      margin-top:20px;
      position: absolute;
        justify-content:center;
        align-items:center;
        position: absolute;
        left: ${left};
        right: ${right};
        top: 90vh;
      li {
        background:white;
        width: 1.2rem;
        height: 1.2rem;
        border-radius: 100%;
        cursor: pointer;
        margin-left:20px;
        &.slick-active {
          background: lightgray;
        }
      }
      button {
        opacity: 0;
        cursor: pointer;
        }
    }
`}
`