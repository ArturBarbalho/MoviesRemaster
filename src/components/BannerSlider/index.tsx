import { settings } from '../../sliderSettings/banner'
import Banner from '../Banner'
import Slider from '../Slider'
import * as S from './styles'
import { BannerProps } from '../../typesProps/index';
import { useEffect, useState } from 'react';
type props = {
  bannerprops:BannerProps[]
}
 
const BannerSlider = ({bannerprops}:props) => {

  //this was necessary because the media querie does not works in the .slick-dots class
  const largura = innerWidth
  const [left,setLeft] = useState('-51%')
  const [right,setRight] = useState('-49%')
  useEffect(()=>{
    if(largura < 500){
    setLeft('-59%')
    setRight('-51%')
  }
  },[])
  
  const slid = bannerprops.map((e)=>{
    return(
       <Banner key={e.title} bannerProps={e} />
       )
  })  
return(
  <S.Wrapper left={left} right={right} >
    <Slider settings={settings}>
    {slid}
    </Slider>
  </S.Wrapper>
)
}
export default BannerSlider
