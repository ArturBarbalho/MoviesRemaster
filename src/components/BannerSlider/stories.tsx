import { Story, Meta } from '@storybook/react/types-6-0'
import BannerSlider from '.'
import { BannerMock } from '../../Mocks/index';

export default {
  title: 'BannerSlider',
  component: BannerSlider
} as Meta

 const parms = [ BannerMock,BannerMock,BannerMock,BannerMock,BannerMock]


export const Default: Story = () => <BannerSlider bannerprops={parms} />