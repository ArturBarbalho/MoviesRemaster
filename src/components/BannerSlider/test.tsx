import { render, screen } from '@testing-library/react'
import '../../../.jest/match-media-mock'
import BannerSlider from '.'
import { BannerMock } from '../../Mocks'
const parms = [ BannerMock,BannerMock,BannerMock,BannerMock,BannerMock]

describe('<BannerSlider />', () => {
  it('should render one slick peer time', () => {
    const { container } = render(<BannerSlider bannerprops={parms} />)

    expect(container.querySelectorAll('li.slick-active')).toHaveLength(1)
    expect(container.firstChild).toMatchSnapshot()
  })
})