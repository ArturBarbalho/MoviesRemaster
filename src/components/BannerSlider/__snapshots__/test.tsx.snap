// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BannerSlider /> should render one slick peer time 1`] = `
<main
  class="styles__Wrapper-sc-15uod80-0 hpFuLL"
>
  <section
    class="styles__Wrapper-sc-12nwagl-0 fTUMGo"
  >
    <div
      class="slick-slider slick-initialized"
      dir="ltr"
    >
      <div
        class="slick-list"
      >
        <div
          class="slick-track"
          style="opacity: 1; transform: translate3d(0px, 0px, 0px);"
        >
          <div
            aria-hidden="true"
            class="slick-slide slick-cloned"
            data-index="-1"
            style="width: 0px;"
            tabindex="-1"
          >
            <div>
              <main
                class="styles__Wrapper-sc-10z8pg5-0"
              >
                <div
                  class="styles__About-sc-10z8pg5-1 cajzIQ"
                >
                  <h1
                    class="styles__Wrapper-sc-1buurg8-0 kNhFRm"
                  >
                     
                     
                    ONYANKOPON O FILME
                    <span>
                      (1800)
                    </span>
                     
                     
                  </h1>
                  <div
                    class="styles__Icon-sc-10z8pg5-2 fXDdrQ"
                  >
                    <img
                      alt="icon1"
                      src="icon1.png"
                    />
                    <img
                      alt="icon2"
                      src="icon2.png"
                    />
                    <img
                      alt="icon3"
                      src="icon3.png"
                    />
                    <img
                      alt="icon4"
                      src="icon4.png"
                    />
                  </div>
                  <p
                    class="styles__Wrapper-sc-mhbed0-0 ldjcec"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis...
                  </p>
                  <button
                    class="styles__Wrapper-sc-11qq8g6-0 hNDJSY"
                    color="black"
                  >
                    See more
                  </button>
                </div>
                <img
                  class="styles__Wrapper-sc-y6e3xk-0 kFeMay"
                  src="https://image.tmdb.org/t/p/original//9VIURlZvfnmFACs2m0MdUDDC3In.jpg"
                />
              </main>
            </div>
          </div>
          <div
            aria-hidden="false"
            class="slick-slide slick-active slick-current"
            data-index="0"
            style="outline: none; width: 0px;"
            tabindex="-1"
          >
            <div>
              <main
                class="styles__Wrapper-sc-10z8pg5-0"
              >
                <div
                  class="styles__About-sc-10z8pg5-1 cajzIQ"
                >
                  <h1
                    class="styles__Wrapper-sc-1buurg8-0 kNhFRm"
                  >
                     
                     
                    ONYANKOPON O FILME
                    <span>
                      (1800)
                    </span>
                     
                     
                  </h1>
                  <div
                    class="styles__Icon-sc-10z8pg5-2 fXDdrQ"
                  >
                    <img
                      alt="icon1"
                      src="icon1.png"
                    />
                    <img
                      alt="icon2"
                      src="icon2.png"
                    />
                    <img
                      alt="icon3"
                      src="icon3.png"
                    />
                    <img
                      alt="icon4"
                      src="icon4.png"
                    />
                  </div>
                  <p
                    class="styles__Wrapper-sc-mhbed0-0 ldjcec"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis...
                  </p>
                  <button
                    class="styles__Wrapper-sc-11qq8g6-0 hNDJSY"
                    color="black"
                  >
                    See more
                  </button>
                </div>
                <img
                  class="styles__Wrapper-sc-y6e3xk-0 kFeMay"
                  src="https://image.tmdb.org/t/p/original//9VIURlZvfnmFACs2m0MdUDDC3In.jpg"
                />
              </main>
            </div>
          </div>
          <div
            aria-hidden="true"
            class="slick-slide"
            data-index="1"
            style="outline: none; width: 0px;"
            tabindex="-1"
          >
            <div>
              <main
                class="styles__Wrapper-sc-10z8pg5-0"
              >
                <div
                  class="styles__About-sc-10z8pg5-1 cajzIQ"
                >
                  <h1
                    class="styles__Wrapper-sc-1buurg8-0 kNhFRm"
                  >
                     
                     
                    ONYANKOPON O FILME
                    <span>
                      (1800)
                    </span>
                     
                     
                  </h1>
                  <div
                    class="styles__Icon-sc-10z8pg5-2 fXDdrQ"
                  >
                    <img
                      alt="icon1"
                      src="icon1.png"
                    />
                    <img
                      alt="icon2"
                      src="icon2.png"
                    />
                    <img
                      alt="icon3"
                      src="icon3.png"
                    />
                    <img
                      alt="icon4"
                      src="icon4.png"
                    />
                  </div>
                  <p
                    class="styles__Wrapper-sc-mhbed0-0 ldjcec"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis...
                  </p>
                  <button
                    class="styles__Wrapper-sc-11qq8g6-0 hNDJSY"
                    color="black"
                  >
                    See more
                  </button>
                </div>
                <img
                  class="styles__Wrapper-sc-y6e3xk-0 kFeMay"
                  src="https://image.tmdb.org/t/p/original//9VIURlZvfnmFACs2m0MdUDDC3In.jpg"
                />
              </main>
            </div>
          </div>
          <div
            aria-hidden="true"
            class="slick-slide"
            data-index="2"
            style="outline: none; width: 0px;"
            tabindex="-1"
          >
            <div>
              <main
                class="styles__Wrapper-sc-10z8pg5-0"
              >
                <div
                  class="styles__About-sc-10z8pg5-1 cajzIQ"
                >
                  <h1
                    class="styles__Wrapper-sc-1buurg8-0 kNhFRm"
                  >
                     
                     
                    ONYANKOPON O FILME
                    <span>
                      (1800)
                    </span>
                     
                     
                  </h1>
                  <div
                    class="styles__Icon-sc-10z8pg5-2 fXDdrQ"
                  >
                    <img
                      alt="icon1"
                      src="icon1.png"
                    />
                    <img
                      alt="icon2"
                      src="icon2.png"
                    />
                    <img
                      alt="icon3"
                      src="icon3.png"
                    />
                    <img
                      alt="icon4"
                      src="icon4.png"
                    />
                  </div>
                  <p
                    class="styles__Wrapper-sc-mhbed0-0 ldjcec"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis...
                  </p>
                  <button
                    class="styles__Wrapper-sc-11qq8g6-0 hNDJSY"
                    color="black"
                  >
                    See more
                  </button>
                </div>
                <img
                  class="styles__Wrapper-sc-y6e3xk-0 kFeMay"
                  src="https://image.tmdb.org/t/p/original//9VIURlZvfnmFACs2m0MdUDDC3In.jpg"
                />
              </main>
            </div>
          </div>
          <div
            aria-hidden="true"
            class="slick-slide"
            data-index="3"
            style="outline: none; width: 0px;"
            tabindex="-1"
          >
            <div>
              <main
                class="styles__Wrapper-sc-10z8pg5-0"
              >
                <div
                  class="styles__About-sc-10z8pg5-1 cajzIQ"
                >
                  <h1
                    class="styles__Wrapper-sc-1buurg8-0 kNhFRm"
                  >
                     
                     
                    ONYANKOPON O FILME
                    <span>
                      (1800)
                    </span>
                     
                     
                  </h1>
                  <div
                    class="styles__Icon-sc-10z8pg5-2 fXDdrQ"
                  >
                    <img
                      alt="icon1"
                      src="icon1.png"
                    />
                    <img
                      alt="icon2"
                      src="icon2.png"
                    />
                    <img
                      alt="icon3"
                      src="icon3.png"
                    />
                    <img
                      alt="icon4"
                      src="icon4.png"
                    />
                  </div>
                  <p
                    class="styles__Wrapper-sc-mhbed0-0 ldjcec"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis...
                  </p>
                  <button
                    class="styles__Wrapper-sc-11qq8g6-0 hNDJSY"
                    color="black"
                  >
                    See more
                  </button>
                </div>
                <img
                  class="styles__Wrapper-sc-y6e3xk-0 kFeMay"
                  src="https://image.tmdb.org/t/p/original//9VIURlZvfnmFACs2m0MdUDDC3In.jpg"
                />
              </main>
            </div>
          </div>
          <div
            aria-hidden="true"
            class="slick-slide"
            data-index="4"
            style="outline: none; width: 0px;"
            tabindex="-1"
          >
            <div>
              <main
                class="styles__Wrapper-sc-10z8pg5-0"
              >
                <div
                  class="styles__About-sc-10z8pg5-1 cajzIQ"
                >
                  <h1
                    class="styles__Wrapper-sc-1buurg8-0 kNhFRm"
                  >
                     
                     
                    ONYANKOPON O FILME
                    <span>
                      (1800)
                    </span>
                     
                     
                  </h1>
                  <div
                    class="styles__Icon-sc-10z8pg5-2 fXDdrQ"
                  >
                    <img
                      alt="icon1"
                      src="icon1.png"
                    />
                    <img
                      alt="icon2"
                      src="icon2.png"
                    />
                    <img
                      alt="icon3"
                      src="icon3.png"
                    />
                    <img
                      alt="icon4"
                      src="icon4.png"
                    />
                  </div>
                  <p
                    class="styles__Wrapper-sc-mhbed0-0 ldjcec"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis...
                  </p>
                  <button
                    class="styles__Wrapper-sc-11qq8g6-0 hNDJSY"
                    color="black"
                  >
                    See more
                  </button>
                </div>
                <img
                  class="styles__Wrapper-sc-y6e3xk-0 kFeMay"
                  src="https://image.tmdb.org/t/p/original//9VIURlZvfnmFACs2m0MdUDDC3In.jpg"
                />
              </main>
            </div>
          </div>
          <div
            aria-hidden="true"
            class="slick-slide slick-cloned"
            data-index="5"
            style="width: 0px;"
            tabindex="-1"
          >
            <div>
              <main
                class="styles__Wrapper-sc-10z8pg5-0"
              >
                <div
                  class="styles__About-sc-10z8pg5-1 cajzIQ"
                >
                  <h1
                    class="styles__Wrapper-sc-1buurg8-0 kNhFRm"
                  >
                     
                     
                    ONYANKOPON O FILME
                    <span>
                      (1800)
                    </span>
                     
                     
                  </h1>
                  <div
                    class="styles__Icon-sc-10z8pg5-2 fXDdrQ"
                  >
                    <img
                      alt="icon1"
                      src="icon1.png"
                    />
                    <img
                      alt="icon2"
                      src="icon2.png"
                    />
                    <img
                      alt="icon3"
                      src="icon3.png"
                    />
                    <img
                      alt="icon4"
                      src="icon4.png"
                    />
                  </div>
                  <p
                    class="styles__Wrapper-sc-mhbed0-0 ldjcec"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis...
                  </p>
                  <button
                    class="styles__Wrapper-sc-11qq8g6-0 hNDJSY"
                    color="black"
                  >
                    See more
                  </button>
                </div>
                <img
                  class="styles__Wrapper-sc-y6e3xk-0 kFeMay"
                  src="https://image.tmdb.org/t/p/original//9VIURlZvfnmFACs2m0MdUDDC3In.jpg"
                />
              </main>
            </div>
          </div>
          <div
            aria-hidden="true"
            class="slick-slide slick-cloned"
            data-index="6"
            style="width: 0px;"
            tabindex="-1"
          >
            <div>
              <main
                class="styles__Wrapper-sc-10z8pg5-0"
              >
                <div
                  class="styles__About-sc-10z8pg5-1 cajzIQ"
                >
                  <h1
                    class="styles__Wrapper-sc-1buurg8-0 kNhFRm"
                  >
                     
                     
                    ONYANKOPON O FILME
                    <span>
                      (1800)
                    </span>
                     
                     
                  </h1>
                  <div
                    class="styles__Icon-sc-10z8pg5-2 fXDdrQ"
                  >
                    <img
                      alt="icon1"
                      src="icon1.png"
                    />
                    <img
                      alt="icon2"
                      src="icon2.png"
                    />
                    <img
                      alt="icon3"
                      src="icon3.png"
                    />
                    <img
                      alt="icon4"
                      src="icon4.png"
                    />
                  </div>
                  <p
                    class="styles__Wrapper-sc-mhbed0-0 ldjcec"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis...
                  </p>
                  <button
                    class="styles__Wrapper-sc-11qq8g6-0 hNDJSY"
                    color="black"
                  >
                    See more
                  </button>
                </div>
                <img
                  class="styles__Wrapper-sc-y6e3xk-0 kFeMay"
                  src="https://image.tmdb.org/t/p/original//9VIURlZvfnmFACs2m0MdUDDC3In.jpg"
                />
              </main>
            </div>
          </div>
          <div
            aria-hidden="true"
            class="slick-slide slick-cloned"
            data-index="7"
            style="width: 0px;"
            tabindex="-1"
          >
            <div>
              <main
                class="styles__Wrapper-sc-10z8pg5-0"
              >
                <div
                  class="styles__About-sc-10z8pg5-1 cajzIQ"
                >
                  <h1
                    class="styles__Wrapper-sc-1buurg8-0 kNhFRm"
                  >
                     
                     
                    ONYANKOPON O FILME
                    <span>
                      (1800)
                    </span>
                     
                     
                  </h1>
                  <div
                    class="styles__Icon-sc-10z8pg5-2 fXDdrQ"
                  >
                    <img
                      alt="icon1"
                      src="icon1.png"
                    />
                    <img
                      alt="icon2"
                      src="icon2.png"
                    />
                    <img
                      alt="icon3"
                      src="icon3.png"
                    />
                    <img
                      alt="icon4"
                      src="icon4.png"
                    />
                  </div>
                  <p
                    class="styles__Wrapper-sc-mhbed0-0 ldjcec"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis...
                  </p>
                  <button
                    class="styles__Wrapper-sc-11qq8g6-0 hNDJSY"
                    color="black"
                  >
                    See more
                  </button>
                </div>
                <img
                  class="styles__Wrapper-sc-y6e3xk-0 kFeMay"
                  src="https://image.tmdb.org/t/p/original//9VIURlZvfnmFACs2m0MdUDDC3In.jpg"
                />
              </main>
            </div>
          </div>
          <div
            aria-hidden="true"
            class="slick-slide slick-cloned"
            data-index="8"
            style="width: 0px;"
            tabindex="-1"
          >
            <div>
              <main
                class="styles__Wrapper-sc-10z8pg5-0"
              >
                <div
                  class="styles__About-sc-10z8pg5-1 cajzIQ"
                >
                  <h1
                    class="styles__Wrapper-sc-1buurg8-0 kNhFRm"
                  >
                     
                     
                    ONYANKOPON O FILME
                    <span>
                      (1800)
                    </span>
                     
                     
                  </h1>
                  <div
                    class="styles__Icon-sc-10z8pg5-2 fXDdrQ"
                  >
                    <img
                      alt="icon1"
                      src="icon1.png"
                    />
                    <img
                      alt="icon2"
                      src="icon2.png"
                    />
                    <img
                      alt="icon3"
                      src="icon3.png"
                    />
                    <img
                      alt="icon4"
                      src="icon4.png"
                    />
                  </div>
                  <p
                    class="styles__Wrapper-sc-mhbed0-0 ldjcec"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis...
                  </p>
                  <button
                    class="styles__Wrapper-sc-11qq8g6-0 hNDJSY"
                    color="black"
                  >
                    See more
                  </button>
                </div>
                <img
                  class="styles__Wrapper-sc-y6e3xk-0 kFeMay"
                  src="https://image.tmdb.org/t/p/original//9VIURlZvfnmFACs2m0MdUDDC3In.jpg"
                />
              </main>
            </div>
          </div>
          <div
            aria-hidden="true"
            class="slick-slide slick-cloned"
            data-index="9"
            style="width: 0px;"
            tabindex="-1"
          >
            <div>
              <main
                class="styles__Wrapper-sc-10z8pg5-0"
              >
                <div
                  class="styles__About-sc-10z8pg5-1 cajzIQ"
                >
                  <h1
                    class="styles__Wrapper-sc-1buurg8-0 kNhFRm"
                  >
                     
                     
                    ONYANKOPON O FILME
                    <span>
                      (1800)
                    </span>
                     
                     
                  </h1>
                  <div
                    class="styles__Icon-sc-10z8pg5-2 fXDdrQ"
                  >
                    <img
                      alt="icon1"
                      src="icon1.png"
                    />
                    <img
                      alt="icon2"
                      src="icon2.png"
                    />
                    <img
                      alt="icon3"
                      src="icon3.png"
                    />
                    <img
                      alt="icon4"
                      src="icon4.png"
                    />
                  </div>
                  <p
                    class="styles__Wrapper-sc-mhbed0-0 ldjcec"
                  >
                    Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis...
                  </p>
                  <button
                    class="styles__Wrapper-sc-11qq8g6-0 hNDJSY"
                    color="black"
                  >
                    See more
                  </button>
                </div>
                <img
                  class="styles__Wrapper-sc-y6e3xk-0 kFeMay"
                  src="https://image.tmdb.org/t/p/original//9VIURlZvfnmFACs2m0MdUDDC3In.jpg"
                />
              </main>
            </div>
          </div>
        </div>
      </div>
      <ul
        class="slick-dots"
        style="display: block;"
      >
        <li
          class="slick-active"
        >
          <button>
            1
          </button>
        </li>
        <li
          class=""
        >
          <button>
            2
          </button>
        </li>
        <li
          class=""
        >
          <button>
            3
          </button>
        </li>
        <li
          class=""
        >
          <button>
            4
          </button>
        </li>
        <li
          class=""
        >
          <button>
            5
          </button>
        </li>
      </ul>
    </div>
  </section>
</main>
`;
